knora/dsp_tools.py

```diff
@@ -7,15 +7,15 @@
 
 import pkg_resources  # part of setuptools
 
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))
+
 from dsplib.utils.onto_create_lists import create_lists
 from dsplib.utils.onto_create_ontology import create_ontology
 from dsplib.utils.onto_get import get_ontology
 from dsplib.utils.onto_process_excel import list_excel2json
 from dsplib.utils.onto_validate import validate_list, validate_ontology
 from dsplib.utils.xml_upload import xml_upload
 
-sys.path.append(os.path.dirname(os.path.realpath(__file__)))
-
```

Comment:
I wonder if that would still be an issue if the main package wasn't called `knora` but `src`, as is convention.

Reply:
however, that would require changes in `setup.py`, so this is for another day I think.